HoneyBadgers.Logic/Services/Searcher.cs

```diff
@@ -10,7 +10,6 @@ public static class Searcher
         public static Dictionary<Movie,int> FindByName(IEnumerable<Movie> movies, string searchInput)
         {
             //TODO: Jak się pozbyć Dictionary?! :o ZAPYTAĆ PATRYK NA PROJEKCJIE, CZY NOWA KLASA TU UJDZIE (Patrz praca domowa 4)
-            searchInput = searchInput;
```

Comment:
Kurna... zapominam cały czas to poprawić. 

`searchInput = searchInput.Trim();`

Tak powinno być

Reply:
😆 
ok, dorzucę do tego poprawkę